app/src/helper/ipfs.ts

```diff
@@ -9,5 +9,10 @@ const ipfs = new IpfsClient({
   protocol: 'https',
 });
 
-// use the ipfs.add method to pin files
+export const saveToIPFS = async (files: any, callback: (hash: string) => void): Promise<void> => {
```

Comment:
What is the purpose of this? Did this not already exist somewhere?

Reply:
It did, just moved it out